Magento/ruleset.xml

```diff
@@ -5,202 +5,438 @@
+        <type>warning</type>
+    </rule>
+    <rule ref="Generic.PHP.DisallowShortOpenTag">
+        <severity>7</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Generic.PHP.DisallowShortOpenTag.EchoFound">
+        <severity>0</severity>
+    </rule>
+    <rule ref="Magento.CodeAnalysis.EmptyBlock">
+        <severity>7</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Magento.Exceptions.Namespace">
+        <severity>7</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Magento.PHP.DiscouragedFunction">
+        <severity>7</severity>
+        <type>warning</type>
+    </rule>
+    <rule ref="Magento.PHP.LiteralNamespaces">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
     <rule ref="Magento.PHP.Var">
-        <severity>8</severity>
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
     <rule ref="Magento.Performance.EmptyCheck">
-        <severity>8</severity>
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Security.IncludeFile">
-        <severity>8</severity>
+    <rule ref="Magento.Strings.StringConcat">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Security.LanguageConstruct">
-        <severity>10</severity>
+    <rule ref="PEAR.Functions.ValidDefaultValue">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Security.Superglobal.SuperglobalUsageError">
-        <severity>10</severity>
+    <rule ref="Squiz.Functions.GlobalFunction">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Security.Superglobal.SuperglobalUsageWarning">
-        <severity>8</severity>
+    <rule ref="Squiz.Operators.IncrementDecrementUsage">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Security.XssTemplate">
-        <include-pattern>*.phtml</include-pattern>
-        <severity>8</severity>
+    <rule ref="Squiz.PHP.GlobalKeyword">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Strings.ExecutableRegEx">
-        <severity>10</severity>
+    <rule ref="Squiz.PHP.NonExecutableCode">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Strings.StringConcat">
-        <severity>8</severity>
+    <rule ref="Squiz.Scope.MemberVarScope">
+        <severity>7</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Templates.ThisInTemplate">
-        <include-pattern>*.phtml</include-pattern>
+
+    <!-- Severity 6 warnings: Code style issues. -->
+    <rule ref="Generic.ControlStructures.InlineControlStructure">
         <severity>6</severity>
+        <type>warning</type>
     </rule>
-    <rule ref="Magento.Translation.ConstantUsage">
+    <rule ref="Generic.Files.ByteOrderMark">
```

Comment:
Propose to increases severity to 9. We can rename section to "Possible security and issues that may cause bugs".